packages/agent/src/agents/graphs/__tests__/interactive.spec.ts

```
@@ -0,0 +1,761 @@
+});
+
+const makeSnakAgent = (overrides: Partial<any> = {}) => ({
+  getAgentConfig: () => ({
+    mode: AgentMode.INTERACTIVE,
+    memory: true,
+    rag: { enabled: true },
+    prompt: { content: 'SYS' },
+  }),
+  getDatabaseCredentials: () => ({ uri: 'postgres://x' }),
+  getMemoryAgent: () =>
+    new (require('../../operators/memoryAgent.js').MemoryAgent)(),
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)